_overviews/scala3-book/domain-modeling-fp.md

```diff
@@ -428,6 +659,9 @@ This can have multiple advantages:
 
 Let us revisit our example once more.
 
+{% tabs module_7 %}
+{% tab 'Scala 3 only' for=module_7 %}
```

Comment:
Ideally this section would be adapted to show how to do the same in scala 2 (i.e. with implicit class)

Reply:
i don't know how to do it. Can I  leave it for capable guy.